### ISSUE TITLE
Error resulting from .DS_Store

### ISSUE THREAD

**Issue by sbacelar** [gohugoio/hugo#5666]
Please ignore `.DS_Store` when reading config files in MacOS. I had to delete `.DS_Store` manually.

**Comment by zwbetz-gh** [comment]
What was the error?

**Comment by sbacelar** [comment]
```
Error: "/Users/sergiobacelar/SB_website/config/_default/.DS_Store:1:1": "/Users/sergiobacelar/SB_website/config/_default/.DS_Store" is not a valid configuration format
To stop the server, run servr::daemon_stop("4324619576") or restart your R session
Serving the directory /Users/sergiobacelar/SB_website at http://127.0.0.1:4321
```

**Comment by tryzniak** [comment]
I think this can be closed because of #5655

**Comment by moorereason** [comment]
> To stop the server, run servr::daemon_stop("4324619576") or restart your R session

None of that is from Hugo.
